src/test/scala/sigmastate/serialization/RelationsSpecification.scala

```diff
@@ -14,8 +14,8 @@ class RelationsSpecification extends TableSerializationSpecification {
       (LE(LongConstant(2), LongConstant(3)), Array[Byte](LeCode, SLong.typeCode, 4, SLong.typeCode, 6)),
       (GT(LongConstant(6), LongConstant(5)), Array[Byte](GtCode, SLong.typeCode, 12, SLong.typeCode, 10)),
       (GE(LongConstant(6), LongConstant(5)), Array[Byte](GeCode, SLong.typeCode, 12, SLong.typeCode, 10)),
-      (EQ(TrueLeaf, FalseLeaf), Array[Byte](EqCode, TrueCode, FalseCode)),
-      (NEQ(TrueLeaf, FalseLeaf), Array[Byte](NeqCode, TrueCode, FalseCode))
+      (EQ(TrueLeaf, FalseLeaf), Array[Byte](EqCode, SBoolean.typeCode, 1, SBoolean.typeCode, 0)),
+      (NEQ(TrueLeaf, FalseLeaf), Array[Byte](NeqCode, SBoolean.typeCode, 1, SBoolean.typeCode, 0))
```

Comment:
@greenhat we need to keep number of bytes minimal. Please consider how to stay within 3 bytes at most

Reply:
@ergomorphic Is it safe to assume that both types in `Relation` sub-types are the same? In this case, I suggest we write a value type only once. As for boolean constants, we can pack them into a bit array.

Reply:
@ergomorphic Oh, I see for `EQ/NEQ` serializer there is a constraint for types to be equal.

Reply:
@greenhat It look like this is hidden non trivial part of removing TrueLeaf objects
Maybe we should return TrueCode back and use them.